attributes/default.rb

```diff
@@ -1,52 +1,50 @@
-# coding: UTF-8
+# encoding: UTF-8
+# Cookbook Name: apache_zookeeper
+# Attributes:: default
+#
+# Apache Zookeeper global attributes
 
-default["zookeeper"]["install_java"] = true
+default['apache_zookeeper']['user'] = 'zookeeper'
+default['apache_zookeeper']['group'] = 'zookeeper'
 
-default["zookeeper"]["user"] = "zookeeper"
-default["zookeeper"]["group"] = "zookeeper"
+default['ulimit']['users'][node['apache_zookeeper']['user']]['filehandle_limit'] = 32768 # ~FC047
+default['ulimit']['users'][node['apache_zookeeper']['user']]['process_limit'] = 1024 # ~FC047
 
-default["ulimit"]["users"][node["zookeeper"]["user"]]["filehandle_limit"] = 32768 # ~FC047
-default["ulimit"]["users"][node["zookeeper"]["user"]]["process_limit"] = 1024 # ~FC047
+default['apache_zookeeper']['env_vars']['ZOO_LOG4J_PROP'] = 'INFO,ROLLINGFILE'
 
-default["zookeeper"]["env_vars"]["ZOO_LOG4J_PROP"] = "INFO,ROLLINGFILE"
+default['apache_zookeeper']['servers'] = []
+default['apache_zookeeper']['follower_port'] = 2888
+default['apache_zookeeper']['election_port'] = 3888
 
-default["zookeeper"]["servers"] = []
-default["zookeeper"]["follower_port"] = 2888
-default["zookeeper"]["election_port"] = 3888
 
-default["zookeeper"]["mirror"] = "http://archive.apache.org/dist/zookeeper"
-default["zookeeper"]["version"] = "3.4.5"
-
-default["zookeeper"]["base_directory"] = "/opt/zookeeper"
-
-default["zookeeper"]["zoo.cfg"]["clientPort"] = 2181
-default["zookeeper"]["zoo.cfg"]["dataDir"] = "/var/zookeeper"
-default["zookeeper"]["zoo.cfg"]["tickTime"] = 2000
-default["zookeeper"]["zoo.cfg"]["autopurge.purgeInterval"] = 24
-default["zookeeper"]["zoo.cfg"]["initLimit"] = 10
-default["zookeeper"]["zoo.cfg"]["syncLimit"] = 5
+default['apache_zookeeper']['zoo.cfg']['clientPort'] = 2181
+default['apache_zookeeper']['zoo.cfg']['dataDir'] = '/var/opt/zookeeper/data'
+default['apache_zookeeper']['zoo.cfg']['tickTime'] = 2000
+default['apache_zookeeper']['zoo.cfg']['autopurge.purgeInterval'] = 24
+default['apache_zookeeper']['zoo.cfg']['initLimit'] = 10
+default['apache_zookeeper']['zoo.cfg']['syncLimit'] = 5
```

Comment:
to address this and data_dir and friend, I'm just going to merge these all back together